symfony/plugins/orangehrmAdminPlugin/Controller/ViewEmailConfigurationController.php

```diff
@@ -20,21 +20,21 @@
 namespace OrangeHRM\Admin\Controller;
 
 use OrangeHRM\Core\Controller\AbstractVueController;
-use OrangeHRM\Core\Service\ConfigService;
-use OrangeHRM\Core\Traits\ServiceContainerTrait;
+use OrangeHRM\Core\Traits\Service\ConfigServiceTrait;
```

Comment:
I prefer `use OrangeHRM\Core\Traits\Service\ConfigService` but not a blocker

Reply:
Can you check this example;
https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php